### PR TITLE
Add is_present? query

### PR DESCRIPTION
**Pull request by backspace**
Hey! While I suggested `is_visible` in #5, I realised I really wanted `is_present`, as I want to know whether the element is on the page, not whether it’s visible. I also called it `is_present?` since it’s a boolean, but if you prefer `is_present` alone, I can change that.

I wasn’t able to use  [`Hound.Matchers.element?/2`](https://hexdocs.pm/hound/Hound.Matchers.html#element?/2) for the scoped query, so I used [`Hound.Helpers.Page.search_within_element/4`](https://hexdocs.pm/hound/Hound.Helpers.Page.html#search_within_element/4) with a `retries` of 1, which works like a scoped `element?`. It’s less elegant band feels a bit hacking, but it permits the same functionality within a collection. What do you think?

### CODE REVIEW

File: lib/queries/is_present.ex [hunk 1]
```diff
@@ -0,0 +1,57 @@
+defmodule PageObject.Queries.IsPresent do
+  @moduledoc """
+    A module wrapper for the is_present? query macro
+  """
+
+  @doc """
+    Defines a module function that determines the present of an element on an html page. The function name is derived
```
Comment (samueljseay) [inline lib/queries/is_present.ex]: just a little typo, should be: "determines the **presence** of an element"

File: test/actions/is_present_test.exs [hunk 2]
```diff
@@ -0,0 +1,41 @@
+defmodule IsPresentPage do
```
Comment (samueljseay) [inline test/actions/is_present_test.exs:1]: Since this is a query specifically for presence, I think it would be worth adding a test for an element that has a style of `display: none;` to confirm it behaves as we expect

**Review by samueljseay — COMMENTED** [review]
Looking great, just a couple little things. Thanks so much for taking the time!

**Comment by backspace** [comment]
Thanks for the review, I made those changes and rebased into one commit. 😀

**Comment by samueljseay** [comment]
Love your work! Thanks!
